dev/config/keycloak/import/ltt-users-0.json

```diff
@@ -0,0 +1,62 @@
+{
+  "realm": "ltt",
+  "users": [
+    {
+      "id": "ltt-test-user",
+      "createdTimestamp": 1709134971000,
```

Comment:
Do we really need to pass in `createdTimestamp` and `createdDate`? I would expect KC to populate it with NOW if we don't...

Reply:
Good idea- however, it looks like when those attributes are omitted, keycloak does not generate them from NOW. Looking at the keycloak API response, `createdTimestamp` is absent and the keycloak UI shows an error (**Invalid Date**):
![image](https://github.com/uwcirg/ltt-environments/assets/5333433/7b09ba09-de8c-47d4-9c08-8d07c49e04cb)


I also tried setting them to `0` (UNIX epoch), but I don't think it's any better since it could be mistaken for a bug:
![image](https://github.com/uwcirg/ltt-environments/assets/5333433/aec01c9c-3979-4fac-91bd-55c9c8709037)

Reply:
Thanks for trying - flagging this for KC gap for @victorlima02 . For the time being I'm certainly fine with what you have.